manager/config/config.go

```diff
@@ -87,8 +87,11 @@ type MysqlConfig struct {
 	// Enable migration
 	Migrate bool `yaml:"migrate" mapstructure:"migrate"`
 
-	// TLS configuration
-	TLS *TLSConfig `yaml:"tls" mapstructure:"tls"`
+    // TLS setting (can be one of "true", "false", "skip-verify",  or "preferred")
+    TLS string `yaml:"tls" mapstructure:"tls"`
```

Comment:
You can refer to harbor, TLSConfig is used to set the SSL mode, do not rename the TLS configuration.

https://github.com/goharbor/harbor/blob/main/src/vendor/github.com/go-sql-driver/mysql/dsn.go#L48

Reply:
Sure good point, fixed in latest commit.